missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/timer/LobbyTimer.java

```diff
@@ -47,14 +47,11 @@ public LobbyTimer(Game game, int startTime) {
 
     @Override
     public void tick() {
-        if (getGame().getPlayers().values().size() == 0) return;
+        if (getGame().getPlayers().size() == 0) return;
 
         for (MWPlayer mp : getGame().getPlayers().values()) {
-            if (mp.getPlayer() != null) {
-                mp.getPlayer().setLevel(seconds);
-            } else {
-                getGame().removePlayer(mp);
```

Comment:
Do we really no longer need this line? This was added as a kind of safeguard, but I don't know if this is needed any longer